packages/gatsby-design-tokens/src/z-indices.js

```diff
@@ -0,0 +1,9 @@
+export default {
+  feedbackWidget: 2,
+  navigation: 5,
+  banner: 10,
+  modal: 10,
+  sidebar: 10,
+  sidebarToggleButton: 20,
```

Comment:
In this case `sidebarToggleButton` also would like to see more general name, but have no idea for it, so will have to live if it will stay, as it is :)

Reply:
Renamed it to `floatingActionButton` (which it indeed is ;-)) in f599bd6.